### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Agathe-Brusset**
Add MailerSend as an email and SMS API provider.

### What change does this PR introduce?

Adding MailerSend to the README, the integration is already finished but wasn't added to the main documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +145,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Telnyx](https://github.com/novuhq/novu/tree/main/providers/telnyx)
 - [x] [Termii](https://github.com/novuhq/novu/tree/main/providers/termii)
 - [x] [Gupshup](https://github.com/novuhq/novu/tree/main/providers/gupshup)
+- [x] [MailerSend](https://github.com/novuhq/novu/tree/main/providers/mailersend)
```
Comment (scopsy) [inline README.md:148]: @Agathe-Brusset I think mailersend only doing email right?

Comment (Agathe-Brusset) [reply README.md:148]: Hey @scopsy ! No, our API also allows sending SMS to US numbers.

Comment (p-fernandez) [reply README.md:148]: Unfortunately, the integration done is just for the email functionalities, so we can't add it to the SMS providers. 😞

Comment (Agathe-Brusset) [reply README.md:148]: Oh okay then, we can drop this line :D

Comment (p-fernandez) [reply README.md]: We will try in the future or with the support of the community to integrate the SMS functionalities, but that might require some work on our side as currently the provider IDs are unique and can't be repeated in integrations for different channels, such this case.

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**